docs/references/account/create-session.md

```diff
@@ -0,0 +1 @@
+Use this endpoint to complete creating the session with a custom session token. Both the **userId** and **secret** arguments will be returned in the response body when sending your request to the [POST /users/:userId/sessions](https://appwrite.io/docs/references/cloud/server-nodejs/users#createToken) endpoint.
```

Comment:
`custom` not needed here anymore, we resue it for all types of tokens.

We should not point directly to createToken .we just say that userId and secret will be returned from request to create a token, or by the flow initiated by token creation (meaning that with phone auth, you get it in your SMS)

Reply:
Updated